src/monocle_apptrace/metamodel/maps/attributes/inference/langchain_entities.json

```diff
@@ -5,30 +5,30 @@
       {
         "_comment": "provider type ,name , deployment , inference_endpoint",
         "attribute": "type",
-        "accessor": "lambda instance,args:'inference.azure_oai'"
+        "accessor": "lambda arguments:'inference.azure_oai'"
       },
       {
         "attribute": "provider_name",
-        "accessor": "lambda instance,args:args['provider_name']"
+        "accessor": "lambda arguments:arguments['kwargs']['provider_name']"
```

Comment:
add data type to arguments

Reply:
Not possible to write data type to arguments in lambda, https://stackoverflow.com/questions/33833881/is-it-possible-to-type-hint-a-lambda-function